queries.sql

```diff
@@ -7,7 +7,39 @@ select name ,escape_attempts from animals where weight_kg > 10.5;
 select * from animals where neutered = true;
 select * from animals where name != 'Gabumon';
 select * from animals where weight_kg >= 10.4 and weight_kg <= 17.3;
+/* Milestone (2)*/
+begin;
+update animals set species ='unspecified';
+select name, species from animals;
+rollback;
 
+begin;
+update animals set species = 'digimon' where name like '%mon';
+update animals set species = 'pokemon' where species is null;
+select name, species from animals;
+commit;
 
 
+begin; 
+delete from animals;
+select name, date_of_birth, neutered, escape_attempts, species  from animals;
+rollback;
+
+begin;
+delete from animals where date_of_birth > 'jan 1, 2022';
+savepoint savepoint_1;
+update animals set weight_kg = weight_kg *-1;
+rollback to savepoint savepoint_1;
+select name, date_of_birth, neutered, escape_attempts, species  from animals;
+update set weight_kg = weight_kg *-1 where weight_kg < 0;
+commit;
```

Comment:
- [ ] Kindly add these transaction queries in the **data.sql** file because they are manipulating data, **queries.sql** file is only responsible for retrieval data queries :blush:

Reply:
Oh ok ... 
even tho i think it was mentioned to put them in queries.sql